licenses/Apache-2.0.lre

```diff
@@ -93,21 +98,24 @@ https://opensource.org/licenses/Apache-2.0
    Definitions.
 
       "License" shall mean the terms and conditions for use, reproduction, and
-      distribution as defined by Sections 1 through 9 of this document.
+      distribution as defined by Sections 1 through 9 of __1__ document.
 
-      "Licensor" shall mean the copyright owner or entity authorized by the
+      (("Licensor" || "Restream"))
+      shall mean the copyright owner or entity authorized by the
       copyright owner that is granting the License.
 
       "Legal Entity" shall mean the union of the acting entity and all other
       entities that control, are controlled by, or are under common control with
-      that entity. For the purposes of this definition, "control" means (i) the
+      that entity. For the purposes of __1__ definition, "control" means (
+      (( i || properties || k ))
```

Comment:
Who has "k"? 
It's pretty weird to have (k), (ii), (iii).

Reply:
https://github.com/apaxa-go/helper/blob/master/LICENSE#L18